tools/sim/bridge.py

```diff
@@ -94,7 +95,7 @@ def cam_callback(self, image):
     yuv_cl = cl_array.empty_like(rgb_cl)
     self.krnl(self.queue, (np.int32(self.Wdiv4), np.int32(self.Hdiv4)), None, rgb_cl.data, yuv_cl.data).wait()
     yuv = np.resize(yuv_cl.get(), np.int32(rgb.size / 2))
-    eof = self.frame_id * 0.05
+    eof = int((time.time() - startTime) * 1000000000)
```

Comment:
I think we still want to set this based on the frame ID. 
```suggestion
    eof = int(self.frame_id * 0.05 * 1e9)
```